cityofaustin/atd-moped
# Update the Team, Tags, and Status filters in the advanced search to use lookup tables

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/17598
https://github.com/cityofaustin/atd-data-tech/issues/18145

This PR:
- updates the **Team** advanced search to default to **contains** operator and support both arbitrary text input or selection from dropdown options
- updates the **Tags** advanced search to default to **contains** operator and support both arbitrary text input or selection from dropdown options
- removes **is** operator from **Team** and **Tags** advanced searches
- adds options to the **Status is** advanced search coming from the distinct values of `phase_name_simple` from the `moped_phases` table

## Testing
**URL to test:** 

https://deploy-preview-1373--atd-moped-main.netlify.app/moped/ or local

**Steps to test:**
1. Test advanced searching with the following fields:
   - Team
      - ~test **Team contains** and then type **m** in the autocomplete and press enter (don't choose from the dropdown). Press **Apply**. You should see results for **Teams contains m**. I'm really open to feedback on how useful this actually is. Do users need to search in this way? I'm on the fence about if this is intuitive or not.~ This functionality was removed
      - test **Team contains** and then start typing your name until you see an option appear that matches. Select the option and then click **Apply**. You should see results for **Team contains < your name >**
   - Tags
      - Repeat the same steps you did for **Team** but with tag values
   - Status
      - test **Status is Potential**, and you should see that this search now has the status options in the dropdown
   - Lead
      - test searching for **Lead is COA ATD** to make sure that the autocomplete still works like it does now in production
      - **is** feels like it should be the default operator for this field. What do y'all think?
   - Full name
      - test searching for **Full name contains test** to make sure the text input search still works like it does in production

---
#### Ship list
